test/kubernetes/e2e/features/access_log/testdata/edge-secure.yaml

```diff
@@ -34,11 +34,11 @@ kind: Gateway
 metadata:
   labels:
     app: gloo
-  name: gateway-proxy
+  name: gateway-proxy-access-log
```

Comment:
Having the same name as the original proxy led to it being deleted during cleanup and causing subsequent tests to fail